targets/ESP32/_nanoCLR/nanoFramework.Hardware.Esp32.Rmt/rmt_multi_stage_encoder.cpp

```
@@ -0,0 +1,210 @@
+
+    return ESP_OK;
+err:
+    // cleanup on error
+    if (multi_encoder)
+    {
+        platform_free(multi_encoder);
+    }
+    return ret;
+}
+
+esp_err_t rmt_add_byte_encoder(rmt_encoder_t *encoder, int loopCount, rmt_bytes_encoder_config_t &config)
```

Cppcheck (2.21.0)

[style] The function 'rmt_add_byte_encoder' is never used.

(unusedFunction)